src/createTerminal.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const readline = require('readline');
+
+function createTerminal() {
```

Comment:
Why do you need to make it as a function? Why not to just return `terminal` as a constant